packages/editor/src/components/block-list/block.js

```diff
@@ -521,7 +537,28 @@ export class BlockListBlock extends Component {
 									isHidden={ ! ( isHovered || isSelected ) || hoverArea !== 'left' }
 								/>
 							) }
-							{ shouldShowContextualToolbar && <BlockContextualToolbar /> }
+
+							{ (
+								shouldShowContextualToolbar ||
+								// When alt + f10 key is pressed we should show the toolbar.
+								this.state.isHandlingAltF10
+							) && (
+								<BlockContextualToolbar
+									// If the toolbar is being shown because it is handling
+									// alt+f10 key press it should focus right after mounting.
+									focusOnMount={ this.state.isHandlingAltF10 }
+								/>
+							) }
+							{ ! shouldShowContextualToolbar && contextualSidebarMayAppear && isSelected && (
+								<KeyboardShortcuts
```

Comment:
An impact of how we render this is that pressing Alt+F10 while in the block inspector will cause focus to be moved into the contextual toolbar. Is that expected?

Reply:
The alternative being to wrap the editable region.

Reply:
> An impact of how we render this is that pressing Alt+F10 while in the block inspector will cause focus to be moved into the contextual toolbar. Is that expected?

That is already the case in master. When we are on the block inspector the toolbar will always be shown normally without the forcing as we are not typing so in this PR we are not changing any behavior for that case. I personally find this behavior acceptable as the block inspector is part of the block.